modules/common.nix

```diff
@@ -353,7 +383,8 @@ in {
       builtins.map (packageWithDerivation: packageWithDerivation.drv or packageWithDerivation.fakeDerivation)
       sortedPackagesWithDerivation
       # types.package calls builtins.storePath
-      ++ sortedStorePaths;
+      ++ sortedStorePaths
+      ++ inlineCapacitorPackages;
```

Comment:
`self` packages are currently added to `manifest.json` and `catalog.json`, so it might be consistent to do the same for these. Honestly though, my preference might be adding neither. If we add everything `catalog.json` is more of a manifest. If we don't add everything, it's more of a lock/publish with invariants that can be recreated anywhere.

Reply:
agreed, in fact i tried, but there are some things missing from the inline packages (eg: `catalogPath`) that i suspect the catalog plugin provides, and thus are not available. Would that be easy to add?

Reply:
`self` packages are just derivations, so I would think it would be exactly the same. `self` packages take this `else` branch https://github.com/flox/floxpkgs/blob/master/modules/common.nix#L276 which calls `readPackage` to add some additional metadata